### ISSUE TITLE
Did you train the models on Audiotime training dataset for reporting the scores in Table 2 of your paper?

### ISSUE THREAD

**Issue by soujanyaporia** [zeyuxie29/AudioCap-Strong#1]
Hi there!

Excellent work. I wonder whether you trained the models e..g, Tango2 on Audiotime train for reporting the scores in Table 2. Could you please confirm?

Thanks.

Best,
Soujanya

**Comment by zeyuxie29** [comment]
Thank you for your interest！

Not yet. The training set of AudioTime was not used to train any models, and the model performance reported in the Table 2 was measured directly on the test set.

**Comment by soujanyaporia** [comment]
Okay, thanks! Great work btw. We have updated Tango 2 paper with STEAM metrics from your paper: https://arxiv.org/pdf/2404.09956
